CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 #### Changed
 - Updated containers based on Debian Jessie to Stretch
 - Support entering state/province for non-US neighborhoods
+- Sort Places by name by default and remove order-by-state option
```

Comment:
Is the capitalization here intentional?

Reply:
I think I erratically capitalize that.  It's not really a term of art in this context, but it's not 100% not one.  It's also closely related to the Neighborhood model, and I do usually capitalize model names.  So my answer is "maybe" :slightly_smiling_face: